README.md

```diff
@@ -3,9 +3,15 @@
 ## Features
 * Webpack Hot Module Reloading (HMR)
 * Optional Webpack Dashboard for useful terminal output while developing
-* A fairly production-ready webpack config (uglify, minify, image compression, image inlining) (tailor it further based on the needs of your project)
+* A fairly production-ready webpack config (dll, uglify, minify, image compression, image inlining) (tailor it further based on the needs of your project)
 
 ## Installation
 * clone the repo
 * npm install
 * npm start to run dev server
+
+## Webpack DLL 
+* this plugin caches vendor dependencies so they're not rebuilt on every hot reload!
```

Comment:
Awesome. This is exactly the kind of stuff that I put off for later and never got around to when making these boilerplates :-)

I guess this serves the same purpose as commons chunk plugin but in a different way, correct?

Reply:
Yes, they are similar. I haven't used commons chunk plugin, but it looks like it can do a wider range of things whereas the DLL plugin just does this one thing well:

https://stackoverflow.com/questions/41890855/webpack-common-chunks-plugin-vs-webpack-dll-plugin

Reply:
An alternative is to cache node modules using cacheDirectory: true for babel-loader: 

https://github.com/babel/babel-loader#options

I think this is how create-react-app handles things